packages/grafana-ui/src/components/Table/utils.ts

```diff
@@ -566,3 +568,23 @@ export function calculateAroundPointThreshold(timeField: Field): number {
 
   return (max - min) / timeField.values.length;
 }
+
+/**
+ * This fires on click of anything in the table, if it's a table cell we grab the row and column
+ *
+ * @param onCellClick
+ * @warning if there's html in the table cell that isn't role=cell this isn't going to callback
```

Comment:
it would be simple enough to traverse up the tree until we find the role=table or role=cell, but that might cause unintended consequences if the cell contains another dataframe or something else with additional markup, so maybe best to start small?